calculator-micro-services/useAlphaFeed.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ -z "$1" ] 
+then
+  echo "installing default glimpse version"
+  version=""
+elif [ $1 = "--help" ] || [ $1 = "--h" ] || [ $1 = "-h" ] || [ $1 = "-?" ]
+then
+  echo "usage: useAlphaFeed.sh <version>"
+  echo "usage: useAlphaFeed.sh 0.21.4"
+  exit 0
+else
+  echo "installing glimpse version $1"
+  version=@$1
+fi
+
+npm config set @glimpse:registry=https://www.myget.org/F/g-alpha/npm/
```

Comment:
Would it make sense to "reset" this back to the default (i.e. NPM feed) after installation is complete?  Otherwise using this script will have the side effect of making *all* installations of Glimpse for *any* other project thereafter use the alpha feed.  Alternatively, maybe just write an `.npmrc` file with the registry setting which then limits use of the feed to the sample project.